esmf_utils/OutputInfo.F90

```diff
@@ -240,11 +257,11 @@ function make_ungridded_dim(info, n, rc) result(ungridded_dim)
       end if
       _ASSERT(is_present, 'Key ' // key // ' not found in ' // trim(json_repr))
```

Comment:
```suggestion
      _ASSERT(is_present, 'Key ' // key // ' not found in ' // trim(json_repr))
```
I thin this assertion should be moved into the conditional on `is_present`.  And then just use `_FAIL` instead of `_ASSERT`

Reply:
I'll take a look.

Reply:
Done.